.dockerignore

```diff
@@ -0,0 +1,30 @@
+**/__pycache__
+**/.classpath
+**/.dockerignore
+**/.env*
+**/.git
+**/.gitignore
+**/.project
+**/.settings
+**/.toolstarget
+**/.vs
+**/.vscode
+**/*.*proj.user
+**/*.dbmdl
+**/*.jfm
+**/azds.yaml
+**/bin
+**/charts
+**/docker-compose*
+**/compose*
+**/Dockerfile*
+**/node_modules
+**/npm-debug.log
+**/obj
+**/secrets.dev.yaml
+**/values.dev.yaml
+README.md
+chromedriver.exe
+gunicorn_logs.txt
+Vagrantfile
```

Comment:
This is a good list but there are a couple of extra things you could ignore

Most important one is the `.venv` virtual environment folder.

You could also ignore e.g. the .vagrant folder that gets created when you create the vagrant VM.